src/main/scala/com/codacy/client/stash/service/ProjectServices.scala

```diff
@@ -0,0 +1,81 @@
+package com.codacy.client.stash.service
+
+import com.codacy.client.stash.client.{Request, RequestResponse, StashClient}
+import com.codacy.client.stash._
+
+class ProjectServices(client: StashClient) {
+
+  val BASE: String = "/rest/api/1.0/projects"
+
+  /**
+    * Only projects for which the authenticated user has the PROJECT_VIEW permission will be returned.
+    */
+  def findAll: RequestResponse[Seq[Project]] = {
+    client executePaginated Request(BASE, classOf[Seq[Project]])
```

Comment:
to be consistent with other parts of the code we should do `client.executePaginated(Request(BASE, classOf[Seq[Project]]))`

Reply:
Good catch